lib/src/screens/restore/widgets/restore_button.dart

```diff
@@ -1,12 +1,11 @@
-import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 
-class RestoreButton extends StatelessWidget {
-  const RestoreButton({
-    required this.onPressed,
-    required this.image,
-    required this.title,
-    required this.description});
+class CardButton extends StatelessWidget {
+  const CardButton(
```

Comment:
the file name is not updated

but anyway, I believe you can revert this renaming since you already created `SupportTile` for the support page